src/migrations/20201016202341-create-legal-information.js

```diff
@@ -15,7 +15,8 @@ module.exports = {
         allowNull: false
       },
       CUIT: {
-        type: Sequelize.INTEGER,
+        type: Sequelize.BIGINT,
```

Comment:
Heads up. You're changing a migration that was already deployed before. This means that the server WILL NOT run this migration again.

Reply:
Yes, as Abel commented, you need to create a new migration in this case.